api/applications/applications_controller_test.go

```diff
@@ -418,6 +418,21 @@ func TestGetApplication_WithEnvironments(t *testing.T) {
 		WithEnvironment("prod", "release").
 		WithEnvironment(anyOrphanedEnvironment, "feature"))
 
+	commonTestUtils.ApplyEnvironment(builders.
+		NewEnvironmentBuilder().
+		WithAppName(anyAppName).WithAppLabel().
+		WithEnvironmentName("dev"))
+
+	commonTestUtils.ApplyEnvironment(builders.
+		NewEnvironmentBuilder().
+		WithAppName(anyAppName).WithAppLabel().
+		WithEnvironmentName("prod"))
+
+	commonTestUtils.ApplyEnvironment(builders.
```

Comment:
This is the only ApplyEnvironment I would have left here

Reply:
I removed them all. I'm not sure what you want to replace with the separate call to ApplyEnvironment now that Application can create it.